Realm/RLMSyncSession.h

```diff
@@ -150,8 +152,20 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (nullable RLMProgressNotificationToken *)addProgressNotificationForDirection:(RLMSyncProgressDirection)direction
                                                                           mode:(RLMSyncProgress)mode
+                                                           dispatchToMainQueue:(BOOL)dispatchToMainQueue
```

Comment:
Oh, I see what you mean now. In that case, I agree. We specify that there's a notification queue, and if they want the work to be done elsewhere they can dispatch it to whatever queue they want. Is that satisfactory?